doc/registrant-api/v1/authentication.md

```diff
@@ -21,7 +21,7 @@ Values in brackets represent values that come from the id card certificate.
 | first_name        | true     | String |                             | Name of the customer (`GN`)                       |
 | last_name         | true     | String |                             | Name of the customer (`SN`)                       |
 | country           | true     | String | 'ee'                        | Code of the country that issued the id card (`C`) |
-| issuing authority | true     | String | 'AS Sertifitseerimiskeskus' |                                                   |
+| issuing_authority | true     | String | 'AS Sertifitseerimiskeskus' |                                                   |
```

Comment:
why not just issuer?

Reply:
Good thought. However, as I'm looking at it, I think it should be dropped completely as we are not accepting any other CAs than SK. 

If needed later for usage with Belgian ID cards, we can restore this parameter as an optional one.

Thoughts?

Reply:
Good idea! +1 for removing it, since it's kinda hardcoded (the only one).